doc/api/buffer.md

```diff
@@ -404,14 +404,14 @@ to initialize a `Buffer` to zeroes.
 Example:
 
 ```js
-const buf = new Buffer(5);
+const buf = new Buffer(10);
```

Comment:
I guess I don't mind the buffer getting a bit bigger, but if you are hoping that means it will always be non-zero.... I don't think that's true. Your own system proves that the initial values can be anything, right? `10` bytes is not special, AFAICT.